features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml

```diff
@@ -179,9 +179,6 @@
 
     <!-- [Organization] Validation Rules API -->
     <Resource context="(.*)/o/api/server/v1/validation-rules(.*)" secured="false" http-method="GET"/>
-    <Resource context="(.*)/o/api/server/v1/validation-rules(.*)" secured="true" http-method="PUT">
```

Comment:
We can remove the above APIs from system-api-resources.xml and system-api-resources.xml.j2 as well

Reply:
already addressed with https://github.com/wso2/carbon-identity-framework/pull/5390